test/test.consumerGroup.js

```diff
@@ -1050,4 +1050,49 @@ describe('ConsumerGroup', function () {
 
     async.series([addMessages, confirmMessages, confirmMessages], done);
   });
+
+  describe('#addTopics', function () {
+    let topic, newTopic, testMessage, consumerGroup;
+
+    before('create the topics', done => {
+      topic = uuid.v4();
+      newTopic = uuid.v4();
+      testMessage = uuid.v4();
+      consumerGroup = new ConsumerGroup({
```

Comment:
can we close the consumerGroup in the `after` ?

Reply:
done